sklearn/ensemble/gradient_boosting.py

```diff
@@ -207,13 +271,61 @@ def predict(self, X):
 
 
 class ScaledLogOddsEstimator(LogOddsEstimator):
-    """Log odds ratio scaled by 0.5 -- for exponential loss. """
+    """Log odds ratio scaled by 0.5 -- for exponential loss.
+
+    This estimator takes only the binary target labels of the training set into
+    account. The prediction is independent of the X values in the training set
+    and those passed to the :func:`predict` function.
+
+    It is like :class:`LogOddsEstimator` but takes the log of the *square* of
+    the odds ratio.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.ensemble import gradient_boosting
+    >>> a = np.random.rand(4, 2)
+    >>> y = np.array([1, 1, 0, 1])
+    >>>
+    >>> scaled = gradient_boosting.ScaledLogOddsEstimator()
+    >>> unscaled = gradient_boosting.LogOddsEstimator()
+    >>> scaled.fit(a, y)
+    >>> unscaled.fit(a, y)
+    >>> print(0.5 * unscaled.prior, scaled.prior)
+    0.5493061443340549 0.5493061443340549
```

Comment:
Please use ellipsis (...) with `# doctest +ELLIPSIS` and keep precisions up to 4 (e.g. `0.1234...`)

Reply:
done